Testing source bedford_gov_uk ...
  found 7 entries for Test_001
    2026-06-04 : Black bin [mdi:trash-can]
    2026-06-04 : Caddy bin [mdi:food-apple]
    2026-06-11 : Caddy bin [mdi:food-apple]
    2026-06-11 : Green bin [mdi:leaf]
    2026-06-11 : Orange bin [mdi:recycle]
    2026-06-18 : Black bin [mdi:trash-can]
    2026-06-18 : Caddy bin [mdi:food-apple]
  Test_002 failed: 404 Client Error: Not Found for url: https://bbaz-as-prod-bartecapi.azurewebsites.net/api/bincollections/residential/getbyuprn/100081207036
    Traceback (most recent call last):
      File "/workspaces/hacs_waste_collection_schedule/custom_components/waste_collection_schedule/waste_collection_schedule/test/test_sources.py", line 135, in test_fetch
        result = source.fetch()
      File "/workspaces/hacs_waste_collection_schedule/custom_components/waste_collection_schedule/waste_collection_schedule/source/bedford_gov_uk.py", line 43, in fetch
        r.raise_for_status()
        ~~~~~~~~~~~~~~~~~~^^
      File "/home/vscode/.local/lib/python3.13/site-packages/requests/models.py", line 1026, in raise_for_status
        raise HTTPError(http_error_msg, response=self)
    requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://bbaz-as-prod-bartecapi.azurewebsites.net/api/bincollections/residential/getbyuprn/100081207036
    
  found 10 entries for Test_003
    2026-06-02 : Black bin [mdi:trash-can]
    2026-06-02 : Caddy bin [mdi:food-apple]
    2026-06-09 : Caddy bin [mdi:food-apple]
    2026-06-09 : Green bin [mdi:leaf]
    2026-06-09 : Orange bin [mdi:recycle]
    2026-06-16 : Black bin [mdi:trash-can]
    2026-06-16 : Caddy bin [mdi:food-apple]
    2026-06-23 : Caddy bin [mdi:food-apple]
    2026-06-23 : Green bin [mdi:leaf]
    2026-06-23 : Orange bin [mdi:recycle]
  found 8 entries for Test_004
    2026-06-05 : Caddy bin [mdi:food-apple]
    2026-06-05 : Green bin [mdi:leaf]
    2026-06-05 : Orange bin [mdi:recycle]
    2026-06-12 : Black bin [mdi:trash-can]
    2026-06-12 : Caddy bin [mdi:food-apple]
    2026-06-19 : Caddy bin [mdi:food-apple]
    2026-06-19 : Green bin [mdi:leaf]
    2026-06-19 : Orange bin [mdi:recycle]
    
    
    should we remove test_002?

--- a/custom_components/waste_collection_schedule/waste_collection_schedule/source/bedford_gov_uk.py
+++ b/custom_components/waste_collection_schedule/waste_collection_schedule/source/bedford_gov_uk.py
@@ -8,7 +8,6 @@ DESCRIPTION = "Source for bedford.gov.uk services for Bedford Borough Council, U
 URL = "https://bedford.gov.uk"
 TEST_CASES = {
     "Test_001": {"uprn": "100080009302"},
-    "Test_002": {"uprn": "100081207036"},
     "Test_003": {"uprn": "100080018481"},
     "Test_004": {"uprn": "100080023672"},
 }

